tests/campaigns/campaign_freq_leveldb.py

```diff
@@ -43,9 +43,37 @@ def __init__(
         self._platform = platform
         self._nb_cpus = self._platform.nb_cpus()
         self._cpu_power = CPUPower(comm_layer=platform.comm)
+        self._saved_governors = None
+        self._saved_frequencies_mhz = None
+
+    def setup(self) -> None:
+        self._saved_governors = [
+            self._cpu_power.get_governor(cpu=cpu) for cpu in range(self._nb_cpus)
+        ]
+        self._saved_frequencies_mhz = [
+            self._cpu_power.get_frequency_mhz(cpu=cpu) for cpu in range(self._nb_cpus)
+        ]
+
+    def teardown(self) -> None:
+        for cpu in range(self._nb_cpus):
+            self._cpu_power.set_frequency(
+                frequency_mhz=self._saved_frequencies_mhz[cpu],
+                cpus=[cpu],
+            )
+        for cpu in range(self._nb_cpus):
+            self._cpu_power.set_governor(governor=self._saved_governors[cpu], cpus=[cpu])
 
     def get_frequency_values(self) -> List[int]:
-        return self._cpu_power.get_frequency_values()
+        value = 1500 * (10**6)
+        try:
+            values = self._cpu_power.get_frequency_values()
+        except ValueError:
+            print(
+                f"[WARNING] Not frequency value found with cpupower, setting to {value} Hz",
```

Comment:
`f"[WARNING] No frequency value found with cpupower, setting to {value} Hz",`

Reply:
```suggestion
                f"[WARNING] No frequency value found with cpupower, setting to {value} Hz",
```

Reply:
@apaolillo You'll need to commit it yourself as I cannot approve the suggested change.